google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -158,7 +162,16 @@ def topic topic_name, project: nil, skip_lookup: nil, async: nil
         ##
         # Creates a new topic.
         #
-        # @param [String] topic_name Name of a topic.
+        # @param [String] topic_name Name of a topic. Required.
+        #   The value can be a simple topic ID (relative name), in which
+        #   case the current project ID will be supplied, or a fully-qualified
+        #   topic name in the form `projects/{project_id}/topics/{topic_id}`.
+        #
+        #   The topic ID (relative name) must start with a letter, and
+        #   contain only letters ([A-Za-z]), numbers ([0-9], dashes (-),
+        #   underscores (_), periods (.), tildes (~), plus (+) or percent
+        #   signs (%). It must be between 3 and 255 characters in length, and
```

Comment:
All these character classes should probably be enclosed in backticks for markdown.

Ditto for two additional cases below.

Reply:
Fixed, thanks!